swarm/storage/localstore/localstore.go

```diff
@@ -134,7 +134,7 @@ func New(path string, baseKey []byte, o *Options) (db *DB, err error) {
 	if o == nil {
 		// default options
 		o = &Options{
-			Capacity: 5000000,
+			Capacity: 30000000,
```

Comment:
use the `const` above instead of hardcoding the number

Reply:
Done